janus_core/src/hpke.rs

```diff
@@ -126,6 +129,12 @@ impl FromStr for HpkePrivateKey {
     }
 }
 
+impl Debug for HpkePrivateKey {
```

Comment:
Any reason not to use `#[derivative(Debug)]` and mark the desired fields `#[derivative(Debug="ignore")]`?